packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -184,7 +184,7 @@ test.describe('Call Composite E2E CallPage [Mobile Only]', () => {
     await loadCallPageWithParticipantVideos(pages);
   });
 
-  test.only('local camera switcher button cycles camera', async ({ pages }, testInfo) => {
+  test('local camera switcher button cycles camera', async ({ pages }, testInfo) => {
```

Comment:
Muahahaha.

Blazingly fast e2e tests.... here they go.

Reply:
lol I caught that too when the tests passed in like 10 seconds 😂